kolibri/plugins/user/assets/src/views/profile-page/index.vue

```diff
@@ -4,46 +4,55 @@
     <ui-alert type="success" @dismiss="resetProfileState" v-if="success">
       {{$tr('success')}}
     </ui-alert>
-    <div class="points">
-      <span class="top">{{ $tr('yourPoints') }}</span>
-      <points-icon class="in-points icon" :active="true"/>
-      <span class="total in-points">{{ $formatNumber(totalPoints) }}</span>
-    </div>
-    <form @submit.prevent="submitEdits">
-
-      <core-textbox
-        v-if="hasPrivilege('username')"
-        class="input-field"
-        :invalid="error"
-        :error="errorMessage"
-        :label="$tr('username')"
-        :value="session.username"
-        disabled
-        autocomplete="username"
-        id="username"
-        type="text" />
-
-      <p v-if="isLearner" class="type">{{ $tr('isLearner') }}</p>
-      <p v-if="isCoach" class="type">{{ $tr('isCoach') }}</p>
-      <p v-if="isAdmin" class="type">{{ $tr('isAdmin') }}</p>
-      <p v-if="isSuperuser" class="type">{{ $tr('isSuperuser') }}</p>
-
-      <core-textbox
-        v-if="hasPrivilege('name') && !isSuperuser"
-        class="input-field"
-        :disabled="busy"
-        :label="$tr('name')"
-        v-model="full_name"
-        autocomplete="name"
-        id="name"
-        type="text" />
+
+    <h3>{{ $tr('role') }}</h3>
+    <p>{{role}}</p>
+
+    <template v-if="!isSuperuser">
+      <h3>{{ $tr('points') }}</h3>
+      <p>
+        <points-icon class="points-icon" :active="true"/>
+        <span class="points-num">{{ $formatNumber(totalPoints) }}</span>
+      </p>
+    </template>
+
+    <template v-if="!canEditUsername">
+      <h3>{{ $tr('username') }}</h3>
+      <p>{{session.username}}</p>
+    </template>
+
+    <template v-if="!canEditName && !isSuperuser">
+      <h3>{{ $tr('name') }}</h3>
+      <p>{{session.full_name}}</p>
+    </template>
+
+    <form v-if="canEditUsername || canEditName" @submit.prevent="submitEdits">
+
+      <template v-if="canEditUsername">
+        <h3>{{$tr('username')}}</h3>
+        <core-textbox
+          :disabled="busy"
+          :invalid="error"
+          :error="errorMessage"
+          :value="session.username"
+          autocomplete="username"
+          type="text" />
+      </template>
+
+      <template v-if="canEditName">
+        <h3>{{$tr('name')}}</h3>
```

Comment:
Space/no space within `{{ }}` is mixed everywhere, but I think it's easier to read with the extra spaces. Could you normalize each `{{ }}` to use that?